Running-Mastodon/Production-guide.md

```diff
@@ -98,22 +98,53 @@ It is recommended to create a special user for mastodon on the server (you could
 
 ## General dependencies
 
+### Ubuntu / Debian
+
     sudo apt-get install imagemagick ffmpeg libpq-dev libxml2-dev libxslt1-dev nodejs file git curl
     curl -sL https://deb.nodesource.com/setup_4.x | sudo bash -
 
     sudo apt-get install nodejs
 
     sudo npm install -g yarn
 
+### Centos / RHEL
+
+    sudo yum install libxml2-devel ImageMagick libxslt-devel git curl nodejs file
+    sudo yum -y install epel-release
+    sudo rpm --import http://li.nux.ro/download/nux/RPM-GPG-KEY-nux.ro
+    sudo rpm -Uvh http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
+    sudo yum -y install ffmpeg ffmpeg-devel
+
+    sudo yum group install "Development tools"
+    curl --silent --location https://rpm.nodesource.com/setup_4.x | sudo bash -
+
+    sudo npm install -g yarn
+
 ## Redis
 
+### Ubuntu / Debian
+
     sudo apt-get install redis-server redis-tools
 
+### Centos / RHEL
+
+    sudo yum install redis rubygem-redis
+
 ## Postgres
 
+### Ubuntu / Debian
+
     sudo apt-get install postgresql postgresql-contrib
 
-Setup a user and database for Mastodon:
```

Comment:
It wasn't removed it just got moved down. It should be more clear now with the newest commit.